include/spatio_temporal_voxel_layer/spatio_temporal_voxel_grid.hpp

```diff
@@ -137,8 +137,8 @@ class SpatioTemporalVoxelGrid
   bool IsGridEmpty(void) const;
 
   // Get time information for clearing
-  double GetDecayShift(const double& time_delta);
-  double GetAcceleratedDecayShift(const double& time_delta, \
+  double GetBaseDurationToDecay(const double& time_delta);
```

Comment:
This is just the standard temporal expiration. Perhaps call it GetTemporalClearingTime?

Reply:
How about `GetTemporalClearingDuration`? My intention is to make it clear to readers that what it returns is the length of time, not a single timestamp (which was what was done before acceleration was implemented)

Reply:
ok